index.bs

```diff
@@ -514,12 +507,23 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=]
 
 <div algorithm>
 To <dfn>respond with an error</dfn> given a [=WebSocket connection=]
-|connection| and an [=error code=] |code|:
+|connection|, |command id| and |error code|:
 
- 1. Issue: Form a valid JSON |errorObject| given |code|.
+ 1. If |command id| is not a number, let |command id| be null.
 
- 2. [=Send a WebSocket message=] comprised of |errorObject| over
-    |connection|.
+ 1. Let |error data| be a new map matching the <code>ErrorResponse</code>
```

Comment:
Is this a https://infra.spec.whatwg.org/#maps?

Unfortunately, Infra's [serialize JSON to bytes](https://infra.spec.whatwg.org/#serialize-json-to-bytes) doesn't actually work, as it takes a JS value. https://github.com/whatwg/infra/issues/159 tracks (I think) adding more stuff to allow serializing Infra values. I've asked there. A TODO somewhere around here would be good.

Reply:
I was wrong and have filed https://github.com/whatwg/infra/issues/336.

Reply:
I tried to fix this a bit. It's definitely not perfect but it's at least not as wrong.